cli/Cargo.toml

```diff
@@ -19,3 +19,4 @@ zip = "2.2.1"
 reqwest = { version= "0.12.9", features = ["blocking"] }
 indicatif = "0.17.9"
 serde = { version = "1.0.216", features = ["derive"] }
+mopro-ffi = {path = "../mopro-ffi"}
```

Comment:
I am still concerned about importing `mopro-ffi`
if we will just use the `constants` (which are pretty simple definitions)

I benchmark the difference between the two
without importing `mopro-ffi` is **9s** to install the CLI
with importing `mopro-ffi` is **36s** to install the CLI

I actually prefer we keep `mopro-ffi` and `cli` independent
and define the `constants` again in `cli`
how do you think? @sifnoc @KimiWu123

Reply:
> I benchmark the difference between the two without importing `mopro-ffi` is **9s** to install the CLI with importing `mopro-ffi` is **36s** to install the CLI

That's huge

> I actually prefer we keep `mopro-ffi` and `cli` independent and define the `constants` again in `cli` how do you think? @sifnoc @KimiWu123

I think it would be fine to add a test case in the github workflow to verify that the two constant definitions are identical.
Would creating a new crate solely for constants be excessive? Both options seem fine to me.

Reply:
I like the idea to create a independent crate, but it's just simple definitions as Vivi said. I agree that we should make things simple. Will add another file in `CLI`.